### ISSUE TITLE
🐛Needed modification and improvements in dark mode

### ISSUE THREAD

**Issue by Ankit-11525** [Ankit-11525/e-commerce_redstore.github.io#97]

**Comment by SandeshGC** [comment]
Here is what I came across:
- The texts is dark and so is the background.

![image](https://user-images.githubusercontent.com/59115123/194760690-f2dc7305-f62a-4aec-a3a2-765f96353474.png)

![image](https://user-images.githubusercontent.com/59115123/194760713-20d58df7-cb01-4a43-a4ff-2bd68caed554.png)

- Some parts still have light background:

![image](https://user-images.githubusercontent.com/59115123/194760746-ab65ebc5-315b-4b23-bc0c-a22b6be4355b.png)

- The cart icon gets hidden in the dark mode:

![image](https://user-images.githubusercontent.com/59115123/194760818-6da8ff98-9c6e-4061-b6bf-f322fc8c13eb.png)

![image](https://user-images.githubusercontent.com/59115123/194760834-04112350-003f-4354-abe8-18bedc93578a.png)

- The prices are invisible in cart page

![image](https://user-images.githubusercontent.com/59115123/194760886-4f49a158-8501-44c2-9adc-b5241b107aa3.png)

- The toggle button is not uniform in all pages.
    in cart, about, products page: 
![image](https://user-images.githubusercontent.com/59115123/194760929-8c41aff2-139f-469a-afae-9f396729fe54.png)

    in home page:
![image](https://user-images.githubusercontent.com/59115123/194760970-4c2c6260-3393-4c1a-9042-1b285e321be7.png)

- There is no toggle button in the Contact page.

**Comment by SandeshGC** [comment]
I would like to add that if I toggle the dark/light mode  while I am in any page, it will not stay dark when I navigate to another page or refresh the same page.

**Comment by Ankit-11525** [comment]
oh thats great!
You are always assigned :)

**Comment by SandeshGC** [comment]
😂 I'm sorry but I would like to wait if anyone's interested. Right now I am working on another issue, let's wait till tomorrow evening, then I will take on this issue.

**Comment by Shivank12batra** [comment]
hey Sandesh! I would love to work on this issue.

**Comment by SandeshGC** [comment]
Yeah sure! @Ankit-11525 will assign you as soon as he can.

**Comment by Ankit-11525** [comment]
Yes , you can start your work but I will assign you by latest tonight because of unavailability of laptop.

**Comment by Shivank12batra** [comment]
![IMG_20221011_223042](https://user-images.githubusercontent.com/84995784/195154755-f346eae8-6f88-477c-9eb2-df2f6336bde9.jpg)

hey Ankit, can you look at this design for navbar in dark mode. Will it be fine to you?

**Comment by Ankit-11525** [comment]
ya it's looking good go for it

**Comment by Aarav238** [comment]
@Ankit-11525 Please check , I have already done this in PR #139

**Comment by Shivank12batra** [comment]
hey, I have already made a pull request fixing the issue.

**Comment by Aarav238** [comment]
Hey @Shivank12batra  , I Have Made changes to each and every page and you can see the other changes are pretty well organised. We  both have done some another changes , hoping that both the PR get approval
